src/visualizePath.js

```diff
@@ -0,0 +1,39 @@
+import { PIXELS_PER_TILE } from './constants';
+
+/* eslint no-param-reassign: ["error", { "ignorePropertyModificationsFor": ["renderer"] }] */
+function createPathSprites(path, renderer, cpt) {
+  renderer.pathSprites = [];
+  const pixelsPerCell = PIXELS_PER_TILE / cpt;
+  path.forEach(cell => {
+    // note: all units in pixels
+    const rect = new PIXI.Graphics();
+    rect.beginFill(0x00ff00).drawRect(
+      cell.x * pixelsPerCell, // x coordinate
+      cell.y * pixelsPerCell, // y coordinate
+      pixelsPerCell, // width
+      pixelsPerCell, // height
+    ).alpha = 0.25;
+    renderer.pathSprites.push(rect);
+  });
+}
+
+function clearRects(renderer) {
+  if (renderer.pathSprites) {
+    renderer.pathSprites.forEach(sprite => {
+      renderer.layers.background.removeChild(sprite);
+    });
+  }
+}
+
+function drawRects(renderer) {
+  renderer.pathSprites.forEach(sprite => {
+    renderer.layers.background.addChild(sprite);
+  });
+}
+
+
+export default function drawPlannedPath(path, renderer, cpt) {
```

Comment:
take out renderer as a param here and all the funcs below because it'll always be `tagpro.renderer`